tests/helm/templates/fortio-cli.yaml

```diff
@@ -177,3 +177,104 @@ spec:
           limits:
             cpu: 500m
             memory: "1G"
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: cli-fortio-tproxy-direct
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: cli-fortio-tproxy-direct
+        version: cli-fortio-tproxy-direct
+    spec:
+      containers:
+      - name: echosrv
+        image: istio/fortio:latest
+        imagePullPolicy: Always
+        args:
+          - load
+          - -t
+          - "0"
+          - -c
+          - "32"
+          - -qps
+          - "{{ .Values.qps }}"
+          - http://fortio-tproxy.test:8080/echo?size=5000
+        resources:
+          requests:
+            cpu: 400m
+            memory: "1G"
+          limits:
+            cpu: 500m
+            memory: "1G"
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: cli-fortio-v1-direct
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: ncli-fortio-v1-direct
+        app: cli-fortio-v1-direct
+        version: cli-fortio-v1-direct
+    spec:
+      containers:
+      - name: echosrv
+        image: istio/fortio:latest
+        imagePullPolicy: Always
+        args:
+          - load
+          - -t
+          - "0"
+          - -c
+          - "32"
+          - -qps
+          - "{{ .Values.qps }}"
+          - http://fortiov1.test:8080/echo?size=5000
+        resources:
+          requests:
+            cpu: 400m
+            memory: "1G"
+          limits:
+            cpu: 500m
+            memory: "1G"
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: cli-fortio-noistio1
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: ncli-fortio-noistio1
+        app: acli-fortio-noistio1
+        version: cli-fortio-noistio1
```

Comment:
shouldn't this have an annotation ?

Reply:
If it did - we would have no metrics (2 pods without istio sidecar calling each other). This way we generate outbound metrics ( and we find that they're not showing up in the dashboard - which is  bug I need to file )